services/community-site/src/pages/Profile/ViewProfile.tsx

```diff
@@ -85,7 +89,30 @@ interface ViewProfileDetailsProps {
 
 function ViewProfileDetails({ points, openEditProfile, openKYCModal }: ViewProfileDetailsProps) {
   const auth = useAuth();
+  const { account } = useMetaMask();
   const history = useHistory();
+  const claimApi = useClaimApi();
+  const [lockedPoints, setLockedPoints] = useState(false);
+  const [calculatedPoints, setCalculatedPoints] = useState<BigNumber>(
```

Comment:
```suggestion
  const [total, setTotal] = useState<BigNumber>(
```
I don't really like this calculated naming approach, it doesn't say much besides that it's calculated 😄